schemas/dab.draft.schema.json

```diff
@@ -239,6 +239,63 @@
             }
           }
         },
+        "mcp": {
```

Comment:
The expected configuration properties in this issue https://github.com/Azure/data-api-builder/issues/2832 are contradictory to what is being done here. 

@JerryNixon can you please confirm if these are the right properties? If they are, issue #2832 needs to be updated to remove the `ai` property as the parent.